cpmonitor/templates/cap_checklist.html

```diff
@@ -50,7 +50,7 @@ <h2 class="accordion-header">
                   <p>
                     <strong>Erklärung:</strong>
                   </p>
-                  <p>{{ values.help_text }}</p>
+                  <p class="pre-formatted-text">{{ values.help_text }}</p>
```

Comment:
The advantage of using this instead of html (<p>) in the help texts is that the formatting in the admin is nicer (not so huge text, as it is already quite long...) But open for discussion.

Reply:
I didn't see a difference in the admin at all, but maybe I wasn't looking for the roght thing...